src/features/WordTable/index.tsx

```diff
@@ -166,6 +178,10 @@ const WordTable = ({
             selectNewCard();
             deleteCards(activeCardIds);
           }}
+          flipCards={() => {
+            selectNewCard();
```

Comment:
hmm I don't think we need this here

Reply:
I agree! But I need to update some state to update the UI

Reply:
why? If the internal card storage value gets updated, the entire UI will update anyways. (see the `useStorage` function in `storage.ts`)

Reply:
oh wait I was testing in the web version rip